cli-toolchain/quickstart.md

```diff
@@ -117,6 +117,50 @@ Do you want to use the default authentication and security configuration?
   I want to learn more.
 ```
 
+#### Adding a Lambda Trigger
+
+There are two ways to setup Lambda Triggers for your Cognito User Pool.
+
+1. In the default Auth CLI workflow, you will be presented with a list of Lambda Trigger templates if you opt to configure advanced settings:
+
+```bash
+$ Do you want to enable any of the following capabilities?
+  ❯ ◯ Custom Auth Challenge with Google reCaptcha
```

Comment:
This text & order has changed now with Dustin's latest update.

Reply:
This is fixed.